src/app/component-library/ui-elements/ago/formatDateAgo.ts

```diff
@@ -4,10 +4,12 @@ import {formatDistanceToNow} from 'date-fns';
  * Formats date as "{n} {timeUnit} ago" or "in {n} {timeUnit}s".
  * @param date date
  */
-export function formatDateAgo(date?: Date | null): string {
+export function formatDateAgo(date?: Date | string | null): string {
```

Comment:
does it make sense to create a type alias : type DateInput = Date | string | null  for readability?